docs/labs/swarm.rst

```diff
@@ -4,9 +4,9 @@
 Flocker with Swarm
 ==================
 
-With the :ref:`Flocker Docker plugin <labs-docker-plugin>`, and with Swarm support for the ``--volume-driver`` option, you can use Flocker together with Docker Swarm.
+With the :ref:`Flocker Docker plugin <docker-plugin>`, and with Swarm support for the ``--volume-driver`` option, you can use Flocker together with Docker Swarm.
 
-First, you need to :ref:`install Flocker <labs-installer>` and the :ref:`Flocker Docker plugin <labs-docker-plugin>`.
+First, you need to :ref:`install Flocker <labs-installer>` and the :ref:`Flocker Docker plugin <docker-plugin>`.
```

Comment:
Likewise, Flocker Plugin for Docker. Also I think we should link this to the plugin installation instructions rather than the introduction.